---
bool success
point_cloud_proc/PlaneObjects plane_objects
